chicago/legistar.py

```diff
@@ -77,44 +81,35 @@ def parseDataTable(self, table):
         rows = table.xpath(".//tr[@class='rgRow' or @class='rgAltRow']")
 
 
-        keys = {}
-        for index, header in enumerate(headers):
-            keys[index] = header.text_content().replace('&nbsp;', ' ').strip()
+        keys = [header.text_content().replace('&nbsp;', ' ').strip()
+                for header in headers]
 
         for row in rows:
-          try:
-            data = defaultdict(lambda : None)
-
-            for index, field in enumerate(row.xpath("./td")):
-                key = keys[index]
-                value = field.text_content().replace('&nbsp;', ' ').strip()
-
-                try:
-                    value = datetime.datetime.strptime(value, self.date_format)
-                    value = value.replace(tzinfo=pytz.timezone(self.timezone))
-
-                except ValueError:
-                    pass
+            try:
+                data = defaultdict(lambda : None)
 
+                for key, field in zip(keys, row.xpath("./td")):
+                    text_content = self._stringify(field)
 
-                # Is it a link?
-                address = None
-                link = field.find('.//a')
+                    if field.find('.//a') is not None :
+                        address = self._get_link_address(field.find('.//a'))
+                        if address :
+                            value = {'label': text_content, 
+                                     'url': address}
+                        else :
+                            value = text_content
+                    else :
+                        value = text_content
 
-                if link is not None:
-                    address = self._get_link_address(link)
-                if address is not None:
-                    value = {'label': value, 'url': address}
+                    data[key] = value
 
-                data[key] = value
+                yield data, keys, row
 
-            yield data, keys, row
-
-          except Exception as e:
-            print('Problem parsing row:')
-            print(etree.tostring(row))
-            print(traceback.format_exc())
-            raise e
+            except Exception as e:
+                print('Problem parsing row:')
+                print(etree.tostring(row))
+                print(traceback.format_exc())
+                raise e
```

Comment:
The key thing here is that I want to see the problematic html. How do you suggest we do this.